src/main/java/io/bitrise/trace/step/InjectTraceTask.java

```diff
@@ -415,11 +416,25 @@ private static void appendTraceDependency(final String appBuildGradlePath, final
      * @param content the content to append.
      * @throws IOException when any I/O error occurs with the file on the path.
      */
-    private static void appendContentToFile(final String path, final String content) throws IOException {
+    private static void appendContent(final String path, final String content) throws IOException {
         logger.debug("Appending to \"{}\" content:\n\"{}\"", path, content);
         Files.write(Paths.get(path), content.getBytes(), StandardOpenOption.APPEND);
     }
 
+    /**
+     * Appends the given content to the top of the given file.
+     *
+     * @param path    the path of the file.
+     * @param content the content to append.
+     * @throws IOException when any I/O error occurs with the file on the path.
+     */
+    private static void appendContentToTop(final String path, final String content) throws IOException {
```

Comment:
nit: can we have a unit test for this method? We've already got a lot of tests for content manipulation - so this would be good to have as well

Reply:
Sure, added test.